egs2/aishell4/enh1/local/musan.name

```diff
@@ -0,0 +1,988 @@
+noise-sound-bible-0017.wav noise-sound-bible
```

Comment:
Both 'audioset.name` and `musan.name` are adapted from ConferencingSpeech: https://github.com/ConferencingSpeech/ConferencingSpeech2021/tree/master/selected_lists/train

But the original files only contain the first column here, i.e., the audio names.
It would be complicated to find out the category information dynamically via a script if I download these files from the ConferencingSpeech repository.
I could not think of a good solution, so I kept these two files here.

Do you have suggestions?

Reply:
Maybe a python script?

Reply:
OK. I have added a script.